common/class.Utils.php

```diff
@@ -207,4 +207,9 @@ public static function toHumanReadablePhpString($value, $indentation = 0)
 
         return (string) $returnValue;
     }
+
+    public static function getUnitId(string $uri): string
```

Comment:
**suggestion**: I would suggest using a more generic name like resource ID or something like that because UnitId implies it is for tests (units).